airbyte-integrations/connectors/source-google-analytics-data-api/setup.py

```diff
@@ -5,13 +5,13 @@
 
 from setuptools import find_packages, setup
 
-MAIN_REQUIREMENTS = ["airbyte-cdk~=0.16", "PyJWT==2.4.0", "cryptography==37.0.4", "requests==2.28.1"]
+MAIN_REQUIREMENTS = ["airbyte-cdk", "PyJWT==2.4.0", "cryptography==37.0.4", "requests"]
```

Comment:
why is this unpinning the version for `requests` and `airbyte-cdk?

Reply:
Since it's not required to be pinned on a particular version and should work with the latest. Any reason why it should be pinned?